examples/clients/python/seafowl/dataframe.py

```diff
@@ -63,7 +70,23 @@ def make_row_inserter(conn: SeafowlConnectionParams) -> RowInserter:
     return partial(query, conn)
 
 
-def dataframe_to_seafowl(
+def export_to_parquet_and_upload(
+    data: pd.DataFrame,
+    conn: SeafowlConnectionParams,
+    destination_table: QualifiedTableName,
+) -> None:
+    url = f"{get_db_endpoint(conn.url, conn.database)}/upload/{destination_table.schema or DEFAULT_SCHEMA}/{destination_table.table}"
+    with tempfile.NamedTemporaryFile(suffix=".parquet") as tmpfile:
+        data.to_parquet(tmpfile, compression="gzip")
+        tmpfile.flush()
+        tmpfile.seek(0)
```

Comment:
Just out of curiosity, is this `seek` needed? Will Seafowl re-use the same file handle?

Reply:
The `requests.post` uses this `tmpfile`, maybe it doesn't rewind automatically.

Reply:
Right, I don't think it rewinds automatically.

Reply:
Understood, thanks!